huggingface/tensorflow/inference/docker/2.6/py3/cu112/Dockerfile.gpu

```diff
@@ -0,0 +1,130 @@
+FROM nvidia/cuda:11.3.1-base-ubuntu20.04
```

Comment:
@philschmid - these are marked as cuda 11.3, but the directory and tagging structure is 11.2. Is there a reason why you used 11.3 here? I'd suggest using 11.2 if possible, since that is in line with the training images